src/components/Vacations.js

```diff
@@ -0,0 +1,30 @@
+import {HiOutlineBookOpen} from 'react-icons/hi';
+
+import useVacationStore from '../hooks/useVacationStore';
+
+function Vacations() {
+	const {vacations, removeVacation} = useVacationStore();
+	return (
+		<>
+			<ul>
+				{vacations.map(vacation => (
+					<li key={vacation.id}>
+						<button>
+							{vacation.vacationName}
+							<HiOutlineBookOpen />
+						</button>
```

Comment:
Why do you render the vacation name as button? Or is the onClick event missing? 🤔